ansible_collections/arista/avd/roles/eos_designs/python_modules/overlay/router_path_selection.py

```diff
@@ -65,6 +65,19 @@ def _get_path_groups(self) -> list:
             if path_group.get("ipsec", True):
                 path_group_data["ipsec_profile"] = ipsec_profile_name
 
+            keepalive = path_group.get("dps_keepalive_interval")
+            if keepalive:
+                if keepalive.get("auto"):
+                    path_group_data["keepalive"] = {"auto": True}
+                else:
+                    interval = keepalive.get("interval")
+                    failure_threshold = get(keepalive, "failure_threshold", default=5)
+                    if interval is not None:
+                        path_group_data["keepalive"] = {
+                            "interval": interval,
+                            "failure_threshold": failure_threshold,
+                        }
```

Comment:
```
keepalive = path_group.get("dps_keepalive_interval", {})
auto = keepalive.get("auto")
interval = keepalive.get("interval")
failure_threshold = get(keepalive, "failure_threshold", default=5)

if auto is not None:
  path_group_data["keepalive"] = {"auto": True}

elif interval is not None:
    path_group_data["keepalive"] = {
        "interval": interval,
        "failure_threshold": failure_threshold,
    }
```

this can help to remove multiple if else condition.